foreman/foreman.spec

```diff
@@ -218,97 +220,150 @@ BuildRequires: http-parser
 # Temporary dep on libuv until https://bugs.centos.org/view.php?id=10606
 # is resolved
 BuildRequires: libuv
-BuildRequires: npm(babel-core) < 6.8.0
-BuildRequires: npm(babel-core) >= 6.7.2
-BuildRequires: npm(babel-loader) < 7.2.0
+BuildRequires: nodejs-packaging
+
+# package.json devDependencies
+BuildRequires: npm(babel-cli) >= 6.10.1
+BuildRequires: npm(babel-cli) < 7.0.0
+BuildRequires: npm(babel-core) >= 6.26.0
+BuildRequires: npm(babel-core) < 7.0.0
+#BuildRequires: npm(babel-eslint) >= 6.1.2
```

Comment:
I commented some dependencies which are in packages.json but I believe should not be needed to build and are test-only deps.